src/modules/auth/screens/login-screen.tsx

```diff
@@ -82,33 +82,53 @@ export const LoginScreen = ({ navigation: { navigate } }: ScreenProps) => {
   };
 
   return (
-    <ImageBackground className='flex-1' resizeMode='stretch' source={images.backgroundAuth()}>
-      <View className='p-8 pt-16 gap-4'>
-        <View className='rounded-lg bg-white px-6 pt-8 pb-4 items-center' >
-          <Image source={images.logoBlackMarket()} className='h-8 w-48'></Image>
-          <View className='w-full mt-8'>
-            <Text variant='body1-small' className='mt-4'>{common.labels.email}</Text>
-            <TextInput autoCapitalize='none' value={email} onChangeText={setEmail} placeholder={common.place_holders.email_input}></TextInput>
-            <Text variant='body1-small' className='mt-4'>{common.labels.password}</Text>
-            <TextInput autoCapitalize='none' value={password} onChangeText={setPassword} secureTextEntry={true} placeholder={common.place_holders.password_input } ></TextInput>
+    <ImageBackground className="flex-1" resizeMode="stretch" source={images.backgroundAuth()}>
+      <View className="gap-4 p-8 pt-16">
+        <View className="items-center rounded-lg bg-white px-6 pb-4 pt-8">
+          <Image source={images.logoBlackMarket()} className="h-8 w-48"></Image>
+          <View className="mt-8 w-full">
+            <Text variant="body1-small" className="mt-4">
+              {common.labels.email}
+            </Text>
+            <TextInput
+              autoCapitalize="none"
+              value={email}
+              onChangeText={setEmail}
+              placeholder={common.place_holders.email_input}></TextInput>
+            <Text variant="body1-small" className="mt-4">
+              {common.labels.password}
+            </Text>
+            <TextInput
+              autoCapitalize="none"
+              value={password}
+              onChangeText={setPassword}
+              secureTextEntry={true}
```

Comment:
You can omit the value if it's `true`:
```suggestion
              secureTextEntry
```

Reply:
done